<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecepcionistaPanel</class>
 <widget class="QMainWindow" name="RecepcionistaPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel del recepcionista</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Panel del recepcionista</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrarCliente">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrar Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCrearOrdenServicio">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crear Orden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrarVehiculo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrar Vehículo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEmitirFacturas">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Emitir Factura</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCerrarSesion">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cerrar Sesión</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
